test/verify/check-metrics

```diff
@@ -1239,6 +1239,15 @@ BEGIN {{
         b.wait_text("[aria-label='Network usage'] [data-interface='cockpittest1'] td[data-label='In']", "0")
         b.wait_text("[aria-label='Network usage'] [data-interface='cockpittest1'] td[data-label='Out']", "0")
 
+        b.assert_pixels(
+            ".current-metrics", "metrics-cards", ignore=[
+                ".pf-c-description-list__text",
+                ".pf-c-progress__measure", ".pf-c-progress__bar",
+                "td[data-label='%']", "td[data-label=In", "td[data-label=Out]",
+                "td[data-label=Used]", "td[data-label=Service]"
+            ]
+        )
```

Comment:
That seems daunting -- pretty much nothing in these cards has a reliable/reproducible contents. But let's see how robust it is.